js/app.js

```diff
@@ -23,14 +23,17 @@ Creature.prototype.render = function(){
 
 Creature.prototype.make_option = function(){
   if ($(`option[value=${this.keyword}]`).length) return;
-  
+
   const new_html = creature_option_template(this);
   $('select').append(new_html);
 }
 
 const get_creature_data = data => {
   $.get(`${data}`, 'json').then(data => {
     data.forEach(val => all_creatures.push(new Creature(val)));
+    all_creatures.sort(function(a,b) { //sort horns

```

Comment:
Placing this here means that the creatures are always sorted by horns. If you want to make it an option selected by the user this code could be part of another function that is called when you receive the user interaction (that also needs to be written and added to the index page).